### PR TITLE
Added Select Text To SetUp Dropdown

### PR DESCRIPTION
**Pull request by glitton**
- Added Select Text to dropdown during the select an event portion of the setup
- Also a bug was identified that makes Wanda cache data makes nerdgraph query return null.  While this isn't fixed yet, added fetchPolicyType: NerdGraphQuery.FETCH_POLICY_TYPE.NO_CACHE to nerdgraph query method.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,12 @@ git --version
 npm -v
 ```
 
+If this is your time building or running a New Relic One application, do the following:
```
Comment (amit-y) [inline README.md]: I think you meant to say "this is your first time building"

Comment (glitton) [reply README.md]: Yes I fixed that, thanks for spotting it.  I plan to add more to the readme from a directions perspective so stay tuned for another PR!

**Comment by glitton** [comment]
Updated master branch to ensure edit board functionality was added.  Checked and it is working.

**Comment by glitton** [comment]
Ensured that edits to rows and columns keep current state of events.

**Pull request merged**